### DISCUSSION TITLE
Self-host in Synology: "container supabase-analytics is unhealthy"

### DISCUSSION THREAD

**Discussion by soleuil** [supabase/supabase#23203] [Questions]
Hello,

I have been trying for some time now to get an instance of Supabase running in my Synology 718+ with docker but I always get the "container supabase-analytics is unhealthy". I have followed the documentation and edited the .env file in the required files.

In order to confirm that it was not something I was doing wrong, I tried to start an instance running locally in my computer with docker using exactly the same procedure and it just worked. No problem at all.

I did notice when I ssh in my Synology things that my be a posible cause:

- The docker version is 20.10.23 and I believe it is not posible to update it.

- I managed to install docker compose version v2.17.2 instead of the v1 that is available.

- The port 5432 is not available when I run the docker compose up -d, so I changed it in my .env file to use 6432 and the error warning dissapeared.

- I had to manually create the /db/data and /storage folders. I did it with mkdir in the ssh terminal.

After all this I run the docker compose up -d and all seems to work fine until it reaches the analytics package and it stops with the unhealthy warning.

I have tried to start the rest of the packages but it does not work and I don't get to see the supabase studio.

I hope someone has achieved it and can help me to sort this out.

Thanks!

**Comment by pedrodaroza** [comment]
Same problem here

**Comment by zunami** [comment]
Same Problem 923+ It's a Synology problem. They changed Docker.

**Comment by aantti** [comment]
Anything in the analytics container logs? Could you possibly check with
`docker compose -f docker-compose.yml logs analytics` ?

**Comment by pocketino** [comment]
Hello,
Same here on DS718+ with Portainer.

Analytics and Kong are not running.
Db is up and running and reachable, studio is Working.

For analytics I feel something is missing in the Db.
The db _supabase has not been creted and the schema _analytice does not exist.

There are Created in the db container at init through SQL files. But is seems all SQL files inside the subdirectory "migrations" have not been executed. I always clean the db/data folder to make sure the init is triggered.

Any idea?
